wishlist.css

```diff
@@ -0,0 +1,42 @@
+.wishlist__page {
+    width: 80%;
+    margin: 30px auto;
+    display: flex;
+}
+
+.wishlist__list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    padding: 10px;
+}
+
+.wishlist__card {
+    border: 1px solid black;
+    width: 230px;
```

Comment:
Avoid using pixels as pixels are absolute unit. You will face many challanges when you try to make it responsive to different viewports.

Reply:
yes will make that change